### PR TITLE
Feature/sk new page/our teachers

### PR DESCRIPTION
**Pull request by SvitlanaKalmutska**

### CODE REVIEW

File: src/MOCKS/our_teachers_info_MOCK.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    imgInfo: {
+      url: 'https://i.imgur.com/nY2JpK0.png',
+      alt: 'Khrystyna Strynadyuk',
+    },
+    teacherInfo: {
+      name: 'Христина Стринадюк',
+      technique: 'Акварель, офорт, ілюстрація',
+    },
+  },
+
```
Comment (tania833) [inline src/MOCKS/our_teachers_info_MOCK.js]: empty

File: src/pages/Artists/OurTeachers.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import styles from './OurTeachers.module.scss';
+import Title from '../../components/TitleOnly';
+import AboutTeachers from './components/AboutTeachers/AboutTeachers';
+import TeachersSearch from './components/TeachersSearch/TeachersSearch';
+import TeachersFooter from './components/TeachersFooter/TeachersFooter';
+import our_teachers_info from '../../MOCKS/our_teachers_info_MOCK';
+
+const OurTeachers = () => {
+  return (
+    <>
```
Comment (tania833) [inline src/pages/Artists/OurTeachers.jsx]: if you have div className={styles.teachersContainer}, which is already a parent-container for other elements, what do you use empty coverage <></> for?

File: src/pages/Artists/OurTeachers.module.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
+@mixin flex($jc, $ai, $fd, $fw) {
```
Comment (tania833) [inline src/pages/Artists/OurTeachers.module.scss:1]: cool

File: src/pages/Artists/components/AboutTeachers/AboutTeachers.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            {item.teacherInfo.technique}
+          </div>
+        </a>
+      </div>
+    </div>
+  ));
+  return <>{items}</>;
+};
+
+AboutTeachers.propTypes = {
```
Comment (tania833) [inline src/pages/Artists/components/AboutTeachers/AboutTeachers.jsx:32]: cool

File: src/pages/Artists/components/AboutTeachers/AboutTeachers.module.scss [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.teachersPhoto__wrapper {
+    @include flex(space-between, center, column, wrap);
+}
+
+.teachersInfo__name {
+    padding-top: 2.3125rem;
+    text-align: center;
+    font-family: Montserrat;
```
Comment (tania833) [inline src/pages/Artists/components/AboutTeachers/AboutTeachers.module.scss]: add quotes

File: src/pages/Artists/components/AboutTeachers/AboutTeachers.module.scss [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    font-size: 1.125rem;
+    font-weight: 500;
+}
+
+.teachersInfo__technique {
+    padding-top: 1.0625rem;
+    text-align: center;
+    font-family: OpenSans;
```
Comment (tania833) [inline src/pages/Artists/components/AboutTeachers/AboutTeachers.module.scss]: "Opensans" please

File: src/pages/Artists/OurTeachers.module.scss (hunk 3 above)
Comment (SvitlanaKalmutska) [reply src/pages/Artists/OurTeachers.module.scss:1]: it's not my )) but I also can :)

**Pull request merged**